build.gradle

```diff
@@ -1,9 +1,18 @@
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
```

Comment:
I've heard somewhere that `jcenter` is faster than `mavenCentral` when it comes to downloading and resolving dependencies.